### PR TITLE
#19 Create and fix auth-v2 test, create create-project test and renam…

### PR DESCRIPTION
**Pull request by Leitirion**
#19 Create and fix auth-v2 test, create create-project test and rename project.

### CODE REVIEW

File: cypress/integration/features/auth-v2/auth-v2.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    // cy.xpath(page.button.locator).click()
+    cy.visit(url)
+    cy.xpath('//a[text()="Login"]').click()
+    cy.xpath("//input[@name='email']").type(emailAddress);
+    cy.xpath("//input[@name='password']").type(password);
+    cy.xpath("//button[@data-id='submit']").click();
+})
+// assert
+Then(`then auth-v2`, () => {
+    cy.xpath('.//div[text()="123@gmail.com"]').should('have.text', emailAddress);
```
Comment (mikementor) [inline cypress/integration/features/auth-v2/auth-v2.js]: //div[@data-id='profile-listbox']//div[@class='text']

File: cypress/integration/features/auth-v2/auth-v2.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Given(`given auth-v2`, () => {
+    // cy.xpath(page.button.locator).click()
+    cy.visit(url)
+    cy.xpath('//a[text()="Login"]').click()
+    cy.xpath("//input[@name='email']").type(emailAddress);
+    cy.xpath("//input[@name='password']").type(password);
+    cy.xpath("//button[@data-id='submit']").click();
+})
+// assert
+Then(`then auth-v2`, () => {
```
Comment (mikementor) [inline cypress/integration/features/auth-v2/auth-v2.js]: `Then profile name is ${name}` ?

File: cypress/integration/features/auth-v2.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+Feature: auth-v2 feature
+Background:
+  Given I authed
+
+  @auth-v2 @focus
+  Scenario: auth-v2 scenario
+    Given given auth-v2
+    Then then auth-v2
```
Comment (mikementor) [inline cypress/integration/features/auth-v2.feature]: `Then profile name is ${name}` ?

File: cypress/integration/features/auth-v2.feature [hunk 4]
```diff
@@ -0,0 +1,9 @@
+
+Feature: auth-v2 feature
+Background:
+  Given I authed
+
+  @auth-v2 @focus
+  Scenario: auth-v2 scenario
+    Given given auth-v2
```
Comment (mikementor) [inline cypress/integration/features/auth-v2.feature]: Given I'm on main page
When I'm sign-in, using email

File: cypress/integration/features/auth-v2.feature [hunk 5]
```diff
@@ -0,0 +1,9 @@
+
+Feature: auth-v2 feature
+Background:
+  Given I authed
```
Comment (mikementor) [inline cypress/integration/features/auth-v2.feature]: -, cause you check it in that test

File: cypress/integration/features/auth-v2.feature [hunk 6]
```diff
@@ -0,0 +1,9 @@
+
+Feature: auth-v2 feature
```
Comment (mikementor) [inline cypress/integration/features/auth-v2.feature]: Feature: Sign-in, using email

File: cypress/integration/features/auth-v2/auth-v2.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import { Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = Cypress.env("homepage");
+const emailAddress = Cypress.env("emailAddress1");
+const password = Cypress.env("password1");
+
+
+
+// arrange
+Given(`given auth-v2`, () => {
```
Comment (mikementor) [inline cypress/integration/features/auth-v2/auth-v2.js]: rename implementation steps according to the steps in .feature file

File: cypress/integration/features/createprojecttest.feature [hunk 8]
```diff
@@ -0,0 +1,9 @@
+
+Feature: createprojecttest feature
+Background:
+  Given I authed
+
+  @createprojecttest @focus
```
Comment (mikementor) [inline cypress/integration/features/createprojecttest.feature]: always

Given
When
Then

Given I'm on main page
When I'm click on create project button
And input 'projectName'
Then modal 'create project' should disappear
And 'projectname' appears in projectList

File: cypress/integration/features/createprojecttest/createprojecttest.js [hunk 9]
```diff
@@ -0,0 +1,35 @@
+
+import { Given,Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = Cypress.env("homepage");
+const emailAddress = Cypress.env("emailAddress1");
+const password = Cypress.env("password1");
+const CPT = Cypress.env("CPT");
```
Comment (mikementor) [inline cypress/integration/features/createprojecttest/createprojecttest.js]: `TestProjectName+${Date.now()}`
NO need in env variables
and use understandable  naming! CPT - wtf is this

File: cypress/integration/features/share-test/share-test.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+    cy.visit(url);
+    cy.xpath("//input[@name='email']").type(emailAddress);
+    cy.xpath("//input[@name='password']").type(password);
+    cy.xpath("//button[@data-id='submit']").click();
+})
+// act
+
+// assert
+Then(`then share-test`, () => {
+    cy.get("a[data-id='share']").click();
```
Comment (mikementor) [inline cypress/integration/features/share-test/share-test.js]: remake

File: cypress/integration/features/auth-v2/auth-v2.js (hunk 2 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2/auth-v2.js]: Fixed

File: cypress/integration/features/auth-v2.feature (hunk 3 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: Fixed

File: cypress/integration/features/auth-v2.feature (hunk 4 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: Fixed

File: cypress/integration/features/auth-v2.feature (hunk 5 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: Fixed

File: cypress/integration/features/auth-v2.feature (hunk 6 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: Fixed

File: cypress/integration/features/auth-v2/auth-v2.js (hunk 7 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2/auth-v2.js]: Fixed

File: cypress/integration/features/auth-v2/auth-v2.js (hunk 1 above)
Comment (Leitirion) [reply cypress/integration/features/auth-v2/auth-v2.js]: Fixed

File: cypress/integration/features/createprojecttest.feature (hunk 8 above)
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

**Comment by Leitirion** [comment]
Given убрал в createprojecttest.feature, так как из-за него + из-за чистки storage базы перед тестами тупо разлогинивало после прохождения auth-v2.

File: cypress/integration/features/createprojecttest/createprojecttest.js (hunk 9 above)
Comment (Leitirion) [reply cypress/integration/features/createprojecttest/createprojecttest.js]: Fixed

File: cypress/integration/features/share-test/share-test.js (hunk 10 above)
Comment (Leitirion) [reply cypress/integration/features/share-test/share-test.js]: Deleted

**Comment by Leitirion** [comment]
Waiting for review.

File: cypress/integration/features/auth-v2.feature [hunk 11]
```diff
@@ -0,0 +1,10 @@
+
+Feature: Sign-in, using email feature
+Background:
+  Given I authed, cause you check it in that test
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: fixed

File: cypress/integration/features/auth-v2/auth-v2.js [hunk 12]
```diff
@@ -0,0 +1,25 @@
+
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = Cypress.env("homepage");
+const emailAddress = Cypress.env("emailAddress1");
+const password = Cypress.env("password1");
+
+Given('I authed, cause you check it in that test',() => {
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/auth-v2/auth-v2.js]: Deleted

File: cypress/integration/features/createprojecttest.feature [hunk 13]
```diff
@@ -0,0 +1,10 @@
+
+Feature: Create test project, using this test feature
+Background:
+  Given I create test project, cause you check it in that test
+
+  @createprojecttest @focus
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/createprojecttest.feature [hunk 14]
```diff
@@ -0,0 +1,10 @@
+
+Feature: Create test project, using this test feature
+Background:
+  Given I create test project, cause you check it in that test
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/createprojecttest.feature [hunk 15]
```diff
@@ -0,0 +1,10 @@
+
+Feature: Create test project, using this test feature
+Background:
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/auth-v2.feature [hunk 16]
```diff
@@ -0,0 +1,8 @@
+
+Feature: Sign-in, using email feature
+
+  @auth-v2 @focus
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/auth-v2.feature]: Fixed

File: cypress/integration/features/createprojecttest.feature [hunk 17]
```diff
@@ -0,0 +1,9 @@
...
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/createprojecttest.feature [hunk 18]
```diff
@@ -0,0 +1,9 @@
...
+Given I am on main page
+When I am sign-in, using email
+  @createprojecttest @focus
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/createprojecttest.feature [hunk 19]
```diff
@@ -0,0 +1,9 @@
...
+  Scenario: Create test project, using this test scenario
+    When I am create project
+    Then Then project name is
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest.feature]: Fixed

File: cypress/integration/features/createprojecttest/createprojecttest.js [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+
+When('I am create project', () => {
+    cy.xpath('.//a[@data-id="projectList"]').click({force:true});
+    cy.xpath('.//div[@data-id="project-list-modal"]//button[@class="ui grey button"]').click({force:true});
+    cy.xpath('.//div[@data-id="create-project-modal"]/div[@class="content"]//input[@name="name"]').type(projectName)
+    cy.xpath('.//div[@data-id="create-project-modal"]/div[@class="actions"]/button[@data-id="createProjectButton"]').click({force:true});
+});
+// act
+// assert
+Then('Then project name is', () => {
```
[the remark this answers is not in the document]
Comment (Leitirion) [reply cypress/integration/features/createprojecttest/createprojecttest.js]: Fixed

**Pull request merged**